app/build.gradle

```diff
@@ -50,7 +50,8 @@ dependencies {
     implementation "androidx.lifecycle:lifecycle-viewmodel-ktx:2.3.1"
     implementation "androidx.lifecycle:lifecycle-runtime-ktx:2.4.0-alpha03"
     implementation "androidx.fragment:fragment-ktx:1.3.6"
+    implementation group: 'com.google.guava', name: 'guava', version: '24.1-jre'
```

Comment:
Why we need this unwanted library?

Reply:
Raghu, classes from this library are used in CollectionsUtils and GenericBaseRecyclerViewAdapter Files

import com.google.common.base.Preconditions
import com.google.common.collect.ImmutableMap.Builder

import com.google.common.collect.Lists

Reply:
I think we are not using that things. keep only needed things

Reply:
Removed library